src/Behavioral.Automation/Services/ListServices.cs

```diff
@@ -100,7 +101,7 @@ public static bool ContainsValues(this IEnumerable<string> actualCollection, Lis
             {
                 if (index > maxIndexForValuesList)
                 {
-                    break;
```

Comment:
if actual collection has more values than expected collection, it is ok, and check result should be = true